setup.py

```diff
@@ -66,7 +66,8 @@
     "invenio-jsonschemas>=1.1.1,<1.2.0",
     "invenio-oaiserver>=1.3.0",
     "invenio-pidstore>=1.2.2,<1.3.0",
-    "invenio-records-rest>=1.7.0,<1.8.0",
+    "invenio-records-rest>=1.8.0,<1.9.0",
+    "invenio-rest>=1.2.4",
```

Comment:
This is redundant I think :)

Reply:
are you sure that invenio-rest 1.2.4 is bottom pinned somewhere?

Reply:
alright :+1: